metricflow-semantics/tests_metricflow_semantics/snapshots/test_sg_resolver_output.py/str/test_complete_set_for_simple_metrics__result.txt

```diff
@@ -726,12 +728,6 @@ account_balance:
   │ user__verification__user__identity_verifications                           │ verification,user              │ METRIC           │ JOINED,METRIC                          │ accounts_source,id_verifications                               │
   │ user__view__user__views                                                    │ view,user                      │ METRIC           │ JOINED,METRIC                          │ accounts_source,views_source                                   │
   │ user__views                                                                │ user                           │ METRIC           │ JOINED,METRIC                          │ accounts_source,views_source                                   │
-  │ user__visit__user__visit_buy_conversion_rate                               │ visit,user                     │ METRIC           │ JOINED,METRIC                          │ accounts_source,buys_source,visits_source                      │
```

Comment:
These should be as a result of changing `account_balance` to a metric, but double checking.

Reply:
Actually, this turned out to be an issue with the input conversion metric  being used for the determination of possible group-by items for a conversion metric. Previously, the conversion measure was ignored, but with the move to conversion metrics, the logic was not retained. This has been addressed in an added commit.